test/helpers/RollbackTestHelper.sol

```diff
@@ -0,0 +1,212 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.30;
+
+import {SimpleContract} from "test/helpers/SimpleContract.sol";
+import {CompoundGovernorHelper} from "test/helpers/CompoundGovernorHelper.sol";
+import {IUpgradeRegressionManager} from "src/interfaces/IUpgradeRegressionManager.sol";
+
+/// @title RollbackTestHelper
+/// @notice Helper functions for testing rollback scenarios with SimpleContract
+contract RollbackTestHelper {
+  SimpleContract public simpleContract;
+  IUpgradeRegressionManager public upgradeRegressionManager;
+
+  constructor(SimpleContract _simpleContract, IUpgradeRegressionManager _upgradeRegressionManager) {
+    simpleContract = _simpleContract;
+    upgradeRegressionManager = _upgradeRegressionManager;
+  }
+
+  /// @notice Generates rollback data for changing the fee and owner of SimpleContract
+  /// @param newFee The new fee to set
+  /// @param newOwner The new owner to set
+  /// @return targets Array of target addresses
+  /// @return values Array of ETH values to send
+  /// @return calldatas Array of encoded function calls
+  /// @return description The description of the rollback
+  function generateRollbackData(uint256 newFee, address newOwner)
```

Comment:
The naming of the parameters on these rollback methods is confusing. While it's a `newFee` and `newOwner` in the context of the rollback itself, normally these would be the _original_ values. I would consider renaming the parameters in these methods to something like `_rollbackFee` and `_rollbackOwner`.